### PR TITLE
Convert all `device_type`s to sentence casing

### LINKED ISSUE #923: Inconsistent names of LED device_type
**Describe the bug**
The `device_type` of `LightEmittingDiodeConfig` is `LightEmittingDiode` (in the `session.py`).
 image

The `device_type` of `LightEmittingDiode` is `Light emitting diode` (in the `devices.py`).
image

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by Sun-flow**
closes #923 

Changed all `device_type` fields that were not using sentence casing to use sentence casing.

### CODE REVIEW

File: src/aind_data_schema/core/session.py [hunk 1]
```diff
@@ -82,7 +82,7 @@ class DetectorConfig(AindModel):
 class LightEmittingDiodeConfig(AindModel):
     """Description of LED settings"""
 
-    device_type: Literal["LightEmittingDiode"] = "LightEmittingDiode"
+    device_type: Literal["Light Emitting Diode"] = "Light Emitting Diode"
```
Comment (dyf) [inline src/aind_data_schema/core/session.py:85]: should probably be "Light emitting diode"

**Review by dyf — CHANGES_REQUESTED** [review]

File: src/aind_data_schema/core/session.py (hunk 1 above)
Comment (Sun-flow) [reply src/aind_data_schema/core/session.py:85]: edit: "changed all `device_type` fields (except one) that were not using sentence casing to use sentence casing."

File: src/aind_data_schema/components/devices.py [hunk 2]
```diff
@@ -567,7 +567,7 @@ class OpenEphysAcquisitionBoard(DAQDevice):
     """Multichannel electrophysiology DAQ"""
 
     # required fields
-    device_type: Literal["Open Ephys Acquisition Board"] = "Open Ephys Acquisition Board"
+    device_type: Literal["Open ephys acquisition board"] = "Open ephys acquisition board"
```
Comment (saskiad) [inline src/aind_data_schema/components/devices.py:570]: Since Open Ephys is an organization, let's have it be "Open Ephys acquisition board"

**Review by saskiad — CHANGES_REQUESTED** [review]
one small fix. Otherwise looks good.

**Review by saskiad — APPROVED** [review]

**Review by saskiad — APPROVED** [review]

**Review by dyf — APPROVED** [review]

**Pull request merged**